programming_examples/basic/passthrough_kernel/Makefile

```
@@ -12,7 +12,7 @@ srcdir := $(shell dirname $(realpath $(firstword $(MAKEFILE_LIST))))
 
 include ${srcdir}/../../makefile-common
 
-device = npu
+devicename = npu
```

Suggesting once but this applies in more cases: 

```suggestion
devicename ?= npu
```